web/src/buddy_list.js

```diff
@@ -80,12 +86,55 @@ export class BuddyList extends BuddyListConf {
         }
 
         const items = this.get_data_from_keys(more_keys);
+        const subscribed_users = [];
```

Comment:
I also use "subscribed  users" language in a few places, maybe this should also be called `users_matching_view`?

Reply:
I think `subscribed_users` seems correct when talking about the set of users themselves, and not the section in the UI where we display users that match the view.